src/components/ColorPalette/colorPalette.css

```diff
@@ -0,0 +1,42 @@
+.color-palette-container{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    width: 20%;
```

Comment:
![image](https://user-images.githubusercontent.com/67889207/167776303-3452568b-8f21-4c33-a161-9db4a0b28107.png)
Giving 20% width it will look like this. you can use min and max-width so it expands and shrinks.